config/graphic

```diff
@@ -107,7 +107,7 @@ get_graphicdrivers() {
   fi
 
   if listcontains "${GRAPHIC_DRIVERS}" "vc4"; then
-    GALLIUM_DRIVERS+=" vc4"
+    GALLIUM_DRIVERS+=" vc4 v3d kmsro"
```

Comment:
The previous RPi's will presumably shift to vc4 at some point, so this should be behind a if $DEVICE = RPi4?

Reply:
enabling v3d and kmsro drivers won't have a negative impact on older RPis (except for wasting a bit of space) and I'd rather keep PROJECT/DEVICE selection out of common config files and scripts like this, if possible